source/css/sass/mixins/_typography.scss

```diff
@@ -67,6 +67,21 @@
   vertical-align: middle;
 }
 
+@mixin _font-variant-position($position) {
```

Comment:
@davidcmoulton Should this be part of the public API? If so, we'll need to remove the `_` and add tests.

Reply:
If it's not just limited to `<sup>` and `<sub>` elements then yes this is public API.